test/site_selection.jl

```diff
@@ -28,3 +29,30 @@ end
     @test all([in(sid, [2, 3]) for sid in prefseedsites[prefseedsites.>0]])
     @test all([in(sid, [2, 3]) for sid in prefshadesites[prefshadesites.>0]])
 end
+
+@testset "Guided site selection without ADRIA ecological model" begin
+    dom = ADRIA.load_domain(joinpath(here, "Example_domain"))
+
+    criteria_df = ADRIA.sample(ex_domain, 5) # get scenario dataframe
```

Comment:
Minor nitpick but I prefer two spaces before an inline comment.
I'm influenced by the Python style guide here - makes the comment stand out a bit more IMO